cli/module_loader.rs

```diff
@@ -704,16 +712,20 @@ impl NpmModuleLoader {
 
   pub fn resolve_req_reference(
     &self,
-    reference: &NpmPackageReqReference,
+    req_ref: &NpmPackageReqReference,
     permissions: &PermissionsContainer,
   ) -> Result<ModuleSpecifier, AnyError> {
+    let package_folder = self
+      .npm_resolver
+      .resolve_pkg_folder_from_deno_module_req(req_ref.req())?;
```

Comment:
Are these calls cached? Ie calling it for the same `req_ref.req()` will compute it from scratch?

Reply:
It should be about the same as before. We can do caching improvements in the future if we see it being slow.